### DISCUSSION TITLE
tor@0.4.7.15-12.5.6 — infected Trojan:Win32/Malgent!MTB by Windows defender.

### DISCUSSION THREAD

**Discussion by meltylang** [ScoopInstaller/Scoop#5665] [General]
How to report likes of this correctly?
Maybe scoop can implement report-shortcut similar to "hash check failed"?

tor-browser 12.5.6 (and also probably 12.5.5) is probably infected too.

**Comment by HUMORCE** [comment]
Scoop does not build or host these files, you should report it to upstream.

For this manifest: the hashes and files provided by upstream, it has passed hash check when it installs properly, unless you use `-s, --skip` option.

I am guessing this is a false positive. for security, you need to decide for yourself.

Related: https://forum.torproject.org/t/new-release-tor-browser-12-5-6/9451
